changelogs/unreleased/2342-brito-rafa

```diff
@@ -0,0 +1 @@
+adding labels on backup CRD for k8s major, minor and git versions
```

Comment:
can you rename this file to `2346-brito-rafa`? We use the PR # in the file name.